lm_eval/tasks/french_bench/french_bench_opus_perplexity.yaml

```
@@ -0,0 +1,22 @@
+group:
+  - french_bench_perplexity
+task: french_bench_opus_perplexity
+dataset_path: manu/opus100-en-fr
+output_type: loglikelihood_rolling
+test_split: test
+fewshot_split: validation
+validation_split: validation
+doc_to_text: ""
```

```suggestion
num_fewshot: 0
doc_to_text: ""
```
Let's hardcode this to num_fewshot 0 since it's a ppl task!